modules/install-upgrade/pages/production-cluster-multiregion.adoc

```diff
@@ -0,0 +1,134 @@
+:activeTopics: 1
+:messageSize: 1000
+:messageThroughput: 100000
+:retentionPolicy: 3600
+:ttlPolicy: 24
+:tieredStoragePolicy: n/a
+:messageReplicationFactor: 3
+:clusterReplicationFactor: 2
+
+= Multi-region georeplication
+
+This page contains sizing and configuration information for a georeplicated VM-based production cluster. +
+Pulsar clusters can be configured to replicate data across multiple regions to maximize availability.
+
+We apply our sizing methodology with the following assumptions: +
+
+* Replication will be *asynchronous* and *will not* use a global Zookeeper.
+* We will replicate *all topics* across two regions.
+* We will assume the same time-to-live and retention policies as the initial Pulsar cluster (the cluster being replicated from).
+
+[#aggregate-worksheet]
+== Workload aggregation worksheet
+
+This initial cluster requirements for the Pulsar cluster described in xref:production-cluster-sizing.adoc[] are as below:
+
+.Workload input characteristics
+[cols=2*,options=header]
+|===
+|*Workload input*
+|*Value*
+
+| Number of active topics
+| {activeTopics}
+
+| Average message size
+| {messageSize} bytes
+
+| Incoming message throughput
+| {messageThroughput} messages per second
+
+| Message retention
+| {retentionPolicy} seconds
+
+| TTL Policy
+| {ttlPolicy} hours
+
+| Tiered storage
+| {tieredStoragePolicy}
+
+| Number of cluster replications
+| {clusterReplicationFactor}
+
+|===
+
+With the aggregated workload characteristics, we apply our methodology to these characteristics to size a production cluster. +
+[source,plain,subs="attributes+"]
+----
+Total message size (raw) =
+{messageReplicationFactor} * // replication factor
+{messageSize} * // average message payload size
+{messageThroughput} * // average message throughput
+({ttlPolicy} * {retentionPolicy}) // TTL and retention period
+  = 25,920,000 MB
+  ≅ 25 TB
+----
+
+.. But wait! We need to account for replicating this cluster across two regions.
+Following our assumptions above, we multiply the total message size by the number of regions we are replicating to, which in this case is {clusterReplicationFactor} regions.
+
+[source,plain,subs="attributes+"]
+----
+Adjusted message size (raw) =
+25 TB * // total message size (raw)
+{clusterReplicationFactor} // replication factor
```

Comment:
comment "// replication factor", I wouldn't call this as "replication factor", this is imply the "# of cluster"